src/main/java/lotto/Model/Util/StaticsUtil.java

```diff
@@ -0,0 +1,82 @@
+package lotto.Model.Util;
+
+import java.util.List;
+import System.Constant;
+import lotto.Model.Domain.Lotto;
+import lotto.Model.Domain.Raffle;
+import lotto.Model.Domain.Statics;
+
+public class StaticsUtil {
+
+    private int totalPrize;
+    private int fifthCount;
+    private int fourthCount;
+    private int thirdCount;
+    private int secondCount;
+    private int firstCount;
+
+    public StaticsUtil() {
+        this.totalPrize = 0;
+        this.fifthCount = 0;
+        this.fourthCount = 0;
+        this.thirdCount = 0;
+        this.secondCount = 0;
+        this.firstCount = 0;
+    }
+
+    public Statics setResult(List<Lotto> lottos, Raffle raffle, int purchaseMoney) {
+        checkResult(lottos, raffle);
+        return new Statics(fifthCount, fourthCount, thirdCount, secondCount, firstCount,
+                calculateRevenue(purchaseMoney, totalPrize));
+    }
+
+    public void checkResult(List<Lotto> lottos, Raffle raffle) {
+        for (Lotto lotto : lottos) {
+            int drawResult = drawResult(lotto, raffle);
+            boolean bonusResult = drawBonusNumberResult(lotto, raffle.getBonusNumber());
+
+            setGradeResult(drawResult, bonusResult);
+        }
+    }
+
+    public float calculateRevenue(int purchaseMoney, int totalPrize) {
+        return ((float) (totalPrize) / purchaseMoney) * 100;
+    }
+
+    public void setGradeResult(int drawResult, boolean bonusResult) {
+        if (drawResult == 3) { // 5등
+            totalPrize += Constant.STATICS_FIFTH_PRIZE.getConstant();
+            fifthCount++;
+        }
+
+        if (drawResult == 4) { // 4등
+            totalPrize += Constant.STATICS_FOURTH_PRIZE.getConstant();
+            fourthCount++;
+        }
+
+        if (drawResult == 5) { // 3등
```

Comment:
잘못된 부분이 느껴집니다!
공 5개와 보너스 번호를 맞춘 것을 테스트 해보셨나요?
예상대로라면 2등과 3등이 동시에 이루어 질 것 같습니다.

3등 같은 경우 아래와 같이 보너스 번호를 맞추지 않아야 하는 조건도 추가해야 할 것 같습니다!
```suggestion
        if (drawResult == 5 && !bonusResult) { // 3등
```

Reply:
그러네요...! 테스트 코드를 꼼꼼하게 작성하지 못한 게 여기서 드러나네요 ㅠㅠ
좋은 지적 감사합니다!